dora/tests/integration/src/test/java/alluxio/client/fs/io/FileInStreamIntegrationTest.java

```diff
@@ -463,220 +411,9 @@ public void positionedReadWithoutCaching() throws Exception {
-            // Expect only first block to be cached, other blocks should be empty in Alluxio
-            for (int i = 0; i < st.getFileBlockInfos().size(); i++) {
-              FileBlockInfo info = st.getFileBlockInfos().get(i);
-              if (i == 0) {
-                achieved = achieved && !info.getBlockInfo().getLocations().isEmpty();
-              } else {
-                achieved = achieved && info.getBlockInfo().getLocations().isEmpty();
-              }
-            }
-            return achieved;
-          } catch (Exception e) {
-            return false;
-          }
-        });
-      } else {
-        Thread.sleep(1000);
-        status = mFileSystem.getStatus(uri);
-        Assert.assertEquals(0, status.getInAlluxioPercentage());
-      }
-    }
-  }
-
-  @Test(timeout = 10000)
-  @LocalAlluxioClusterResource.Config(
-          confParams = {PropertyKey.Name.WORKER_BLOCK_HEARTBEAT_INTERVAL_MS, "2000"})
-  public void asyncCacheAfterSeek() throws Exception {
-    String filename = mTestPath + "/file_" + MAX_LEN + "_" + mWriteUnderStore.hashCode();
-    AlluxioURI uri = new AlluxioURI(filename);
-
-    for (ReadType readType : ReadType.values()) {
-      mFileSystem.free(uri);
-      CommonUtils.waitFor("No in-Alluxio data left from previous iteration.", () -> {
-        try {
-          URIStatus st = mFileSystem.getStatus(uri);
-          return st.getInAlluxioPercentage() == 0;
-        } catch (Exception e) {
-          return false;
-        }
-      });
-      FileInStream is = mFileSystem.openFile(uri,
-          OpenFilePOptions.newBuilder().setReadType(readType.toProto()).build());
-      URIStatus status = mFileSystem.getStatus(uri);
-      is.seek(status.getBlockSizeBytes() + 1);
-      is.read();
-      status = mFileSystem.getStatus(uri);
-      Assert.assertEquals(0, status.getInAlluxioPercentage());
-      is.close();
-      if (readType.isCache()) {
-        CommonUtils.waitFor("Second block to be cached.", () -> {
-          try {
-            URIStatus st = mFileSystem.getStatus(uri);
-            boolean achieved = true;
-            // Expect only second block to be cached, other blocks should be empty in Alluxio
-            for (int i = 0; i < st.getFileBlockInfos().size(); i++) {
-              FileBlockInfo info = st.getFileBlockInfos().get(i);
-              if (i == 1) {
-                achieved = achieved && !info.getBlockInfo().getLocations().isEmpty();
-              } else {
-                achieved = achieved && info.getBlockInfo().getLocations().isEmpty();
-              }
-            }
-            return achieved;
-          } catch (Exception e) {
-            return false;
-          }
-        });
-      } else {
-        Thread.sleep(1000);
-        status = mFileSystem.getStatus(uri);
-        Assert.assertEquals(0, status.getInAlluxioPercentage());
-      }
-    }
-  }
-
-  @Test(timeout = 10000)
-  public void asyncCacheFirstBlockPRead() throws Exception {
```

Comment:
async cache feature has been deprecated and removed

Reply:
let's leave this comment open here so if anyone traces the code to this PR they will know what's going on